clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSContext.scala

```diff
@@ -66,6 +66,9 @@ object LakeFSContext {
   val LAKEFS_CONF_DEBUG_GC_MAX_COMMIT_ISO_DATETIME_KEY = "lakefs.debug.gc.max_commit_iso_datetime"
   val LAKEFS_CONF_DEBUG_GC_MAX_COMMIT_EPOCH_SECONDS_KEY = "lakefs.debug.gc.max_commit_epoch_seconds"
   val LAKEFS_CONF_DEBUG_GC_REPRODUCE_RUN_ID_KEY = "lakefs.debug.gc.reproduce_run_id"
+
+  //  Objects that are written during the skip period are not collected
+  val LAKEFS_CONF_DEBUG_GC_UNCOMMITTED_SKIP_SECONDS_KEY = "lakefs.debug.gc.uncommitted_skip_seconds"
```

Comment:
Did you choose this resolution for the tests sake?

Reply:
Yap, do you have anything else in mind?